File: get_feature_153.py
import sqlite3

conn = sqlite3.connect('/Users/peterryszkiewicz/Repos/sus-repo-finder/features.db')
c = conn.cursor()
c.execute('SELECT id, name, description, steps FROM features WHERE id = 153')
row = c.fetchone()
if row:
    print(f'ID: {row[0]}')
    print(f'Name: {row[1]}')
    print(f'Description: {row[2]}')
    print(f'Steps: {row[3]}')
else:
    print('Feature not found')
conn.close()

File: get_feature_153.js
const Database = require('better-sqlite3');
const db = new Database('/Users/peterryszkiewicz/Repos/sus-repo-finder/features.db');
const row = db.prepare('SELECT id, name, description, steps FROM features WHERE id = 153').get();
if (row) {
    console.log('ID:', row.id);
    console.log('Name:', row.name);
    console.log('Description:', row.description);
    console.log('Steps:', row.steps);
} else {
    console.log('Feature not found');
}
db.close();
